gfx/common/wayland_common.c

```diff
@@ -33,6 +33,10 @@
 #define APP_ID "org.libretro.RetroArch"
 #define WINDOW_TITLE "RetroArch"
 
+#ifndef XDG_TOPLEVEL_STATE_SUSPENDED
```

Comment:
This is defined in the header generated by `wayland-scanner` from `xdg-shell.xml` when`./configure` is run.
It's more likely to hide issues than fix them.

Reply:
Removed

Reply:
I meant that the `#define XDG_TOPLEVEL_STATE_SUSPENDED` was unnecessary.
However trying it on my machine I see that `XDG_TOPLEVEL_STATE_SUSPENDED` isn't being defined in `./gfx/common/wayland/xdg-shell.h` because `./gfx/common/wayland/generate_wayland_protos.sh` is selecting my distros protocol files which are not as up to date.

Reply:
I have increased requirement `wayland-protocols` version to 1.32, so if it's not found - bundled (and updated) `xdg-shell` will be used instead.

Reply:
Hah, I completely missed that.
I was trying to hack pkg-config support into `./gfx/common/wayland/generate_wayland_protos.sh`.